### PR TITLE
BASE CSS #28

### PR DESCRIPTION
**Pull request by Arshunas**
Signed-off-by: Sarunas <sarunas.stankevicius@mailbox.org>

### CODE REVIEW

File: css/base/base.css [hunk 1]
```diff
@@ -15,5 +15,18 @@ body {
...
+p > a {
+    color: red;
+    text-decoration: none;
+}
+p > a:hover {
+    text-decoration: none;
+}
+
 /* base css start #28 */
 /* base css end */
```
Comment (vitgri1) [inline css/base/base.css:32]: kodas turetu buti tarp komentaru

File: css/base/base.css [hunk 2]
```diff
@@ -15,5 +15,18 @@ body {
...
+    font-size: 16px;
+    line-height: 1.2em;
+    color: #333;
+}
+p > a {
+    color: red;
+    text-decoration: none;
+}
+p > a:hover {
+    text-decoration: none;
```
Comment (vitgri1) [inline css/base/base.css:28]: mes norim at visu linku, ne tik p>a

**Review by vitgri1 — CHANGES_REQUESTED** [review]

File: css/base/base.css [hunk 3]
```diff
@@ -16,4 +16,19 @@ body {
...
+    line-height: 1.2em;
+    color: #222222;
+}
+p > a {
+    color: #222222;
+    text-decoration: none;
+}
+a:hover {
+    text-decoration: none;
+}
```
Comment (vitgri1) [inline css/base/base.css:30]: No

File: css/base/base.css [hunk 4]
```diff
@@ -16,4 +16,19 @@ body {
...
+/* base css end */
+
+
+
```
Comment (vitgri1) [inline css/base/base.css:34]: Random extra lines

**Review by vitgri1 — CHANGES_REQUESTED** [review]

File: css/base/base.css [hunk 5]
```diff
@@ -16,4 +16,19 @@ body {
 }
 
 /* base css start #28 */
-/* base css end */
\ No newline at end of file
+p {
+    font-size: 16px;
+    line-height: 1.2em;
+    color: #222222;
```
Comment (vitgri1) [inline css/base/base.css:22]: Random color

File: css/base/base.css [hunk 6]
```diff
@@ -16,4 +16,19 @@ body {
...
+}
+p > a {
```
Comment (vitgri1) [inline css/base/base.css:24]: per didelis prioritetas/ tik atvejai kai 
```
<p>text <a>link<a/> </p>
```

File: css/base/base.css [hunk 7]
```diff
@@ -16,4 +16,19 @@ body {
...
+    color: #222222;
```
Comment (vitgri1) [inline css/base/base.css:25]: random color

File: css/base/base.css [hunk 8]
```diff
@@ -16,4 +16,19 @@ body {
...
+    text-decoration: none;
+}
+a:hover {
+    text-decoration: none;
```
Comment (vitgri1) [inline css/base/base.css:29]: jau aprasyta

File: css/base/base.css [hunk 9]
```diff
@@ -16,4 +16,19 @@ body {
...
+p {
+    font-size: 16px;
+    line-height: 1.2em;
+    color: #222222;
+}
+p > a {
+    color: #222222;
+    text-decoration: none;
+}
+a:hover {
```
Comment (vitgri1) [inline css/base/base.css:28]: truksta spalvos keitimo

**Review by vitgri1 — CHANGES_REQUESTED** [review]

**Review by vitgri1 — APPROVED** [review]
LGTM

**Review by inga911 — APPROVED** [review]

**Pull request merged**